docs/examples.rst

```diff
@@ -31,14 +43,29 @@ Just a few examples to get you an idea of how easy PyScaffold is to use:
   and ``.cirrus.yml`` for `Tox <http://tox.testrun.org/>`_ and
   `Cirrus CI <https://cirrus-ci.org/>`_.
 
-``putup my_zope_subpackage --namespace zope -l GPL-3.0-or-later``
-  This will create a project and subpackage named *my_zope_subpackage* in the
-  namespace *zope*. To be honest, there is really only the `Zope project <http://www.zope.org/>`_
+``putup my_zope_subpackage --name my-zope-subpackage --namespace zope --package subpackage``
+  This will create a project under the ``my_zope_subpackage`` directory with
+  the *installation name* of ``my-zope-subpackage`` (this is the name used by
+  pip_ and PyPI_), but with the following corresponding import statement::
+
+    from zope import subpackage
+    # zope is the namespace and subpackage is the package name
+
+  To be honest, there is really only the `Zope project <http://www.zope.org/>`_
   that comes to my mind which is using this exotic feature of Python's packaging system.
   Chances are high, that you will never ever need a namespace package in your life.
```

Comment:
Great comment! I think statements like that help users a lot in figuring out which features are important for them.

Reply:
Hi @FlorianWilhelm thanks for the comment, the original idea of mentioning Zope is yours 😝, I added the code example with the namespaces to make it more concrete...

Reply:
But the sentence `Chances are high, that you will never ever need a namespace package in your life.` is from you right?  That's the one I was referring to.